src/wallet/adapters/ConnectAdaptor.ts

```diff
@@ -38,15 +40,17 @@ export class ConnectAdaptor implements AUTHAdapter {
   constructor({
     chainId,
     apiKey,
+    disableLocal = false,
```

Comment:
I would have added this param at last position.
This way if somebody want to specify  passkeyName, it would not have to pass the value for disable local.